src/Stratis.Bitcoin.IntegrationTests/API/ApiSpecification.cs

```diff
@@ -35,5 +35,127 @@ public void Listmethods_via_rpc_returns_non_empty_list()
             When(calling_rpc_listmethods);
             Then(a_full_list_of_available_commands_is_returned);
         }
+
+        [Fact]
+        public void Block_with_valid_hash_via_api_returns_transaction_block()
+        {
+            Given(two_connected_pow_nodes_with_api_enabled);
+            And(a_block_is_mined_creating_spendable_coins);
+            And(more_blocks_mined_past_maturity_of_original_block);
+            And(a_real_transaction);
+            And(the_block_with_the_transaction_is_mined);
+            When(calling_block_with_valid_hash_via_api_returns_block);
+            Then(the_real_block_should_be_retrieved);
+            And(the_block_should_contain_the_transaction);
+        }
+
+        [Fact]
+        public void Getblockcount_via_api_returns_tipheight()
+        {
+            Given(a_pow_node_with_api_enabled);
+            And(a_block_is_mined_creating_spendable_coins);
+            And(more_blocks_mined_past_maturity_of_original_block);
+            When(calling_getblockcount_via_api_returns_an_int);
```

Comment:
second part of this sentence kinda weird, I think just `calling_getblockcount` is good enough